interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/emails/telehealth-invitation-existing.html.twig

```diff
@@ -0,0 +1,162 @@
+                                        <tr>
+                                            <td height="8" style="height:8px; font-size:0px; line-height:0px;">
+                                                &nbsp;</td>
+                                        </tr>
+                                        <tr>
+                                            <td class="em_grey em_center" align="left" valign="top"
+                                                style="font-family: Arial, sans-serif; font-size: 16px; line-height: 24px; color:#434343;">
+                                                <p>
+                                                    {{ "You've been invited to join a medical telehealth conferencing session."|xlt}}
+                                                </p>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td height="16" style="height:16px; font-size:1px; line-height:1px;">
+                                                &nbsp;</td>
+                                        </tr>
+
+                                        <tr>
+                                            <td align="left" valign="top">
+                                                <table width="140" border="0" cellspacing="0" cellpadding="0"
+                                                       align="left" style="width:140px;" class="em_wrapper">
+                                                    <tr>
+                                                        <td valign="top" align="center">
+                                                            <table width="140"
+                                                                   style="width:160px; background-color:#45AD17; border-radius:4px;"
+                                                                   border="0" cellspacing="0" cellpadding="5"
+                                                                   align="center" bgcolor="#45AD17">
+                                                                <tr>
+                                                                    <td class="em_white" height="34" align="center"
+                                                                        valign="middle"
+                                                                        style="font-family: Arial, sans-serif; font-size: 13px; color:#ffffff; font-weight:bold; height:34px;">
+                                                                        {# TODO: @brady.miller - using attr_url here breaks on MailGun and Thunderbird, any suggestions here? #}
+                                                                        <a href="{{url}}" target="_blank"
+                                                                           style="text-decoration:none; color:#ffffff; line-height:34px; display:block;">
```

Comment:
@bradymiller just want to specifically call this out on how attr_url was actually breaking everything in the email clients.  Do you have any thoughts on this?

Reply:
url escaping meant for just the parameter variables. If you escape an entire url then will break it if it has GET parameters since will escape the `?&` characters. So in the case for href above, would just use `attr`

Reply:
So attr will escape the ?& characters as well.  
```
// .....
 * @param string $text The string to escape, possibly including (&), (<),
 *                     (>), ('), and (").
 * @return string The string, with (&), (<), (>), ("), and (') escaped.
 */
function attr($text)
{
    return htmlspecialchars(($text ?? ''), ENT_QUOTES);
}
```

Reply:
I went ahead and broke up the url into specific pieces that way we can handle the escaping properly.

Reply:
attr escapes them for html though, so they won't muck things up in the url path (ie. the escaped & in attr basically gets changed back to & when used in url).

url however will break things since it is changed to something that will no longer be recognized as a & characters that is used to separate get parameters (this is basically used so that if there is a & or ? character as part of a get parameter (for example, if a get parameter labeled string was equal to hey&man, then it won't break when placed in the url) it won't break things)

Reply:
code looks good!